src/scrollNav.js

```diff
@@ -143,7 +144,7 @@
       var $nav_list = $('<ol />', {'class': S.settings.className + '__list'});
 
       $.each(sections, function(i) {
-        var $item     = (i === 0) ? $('<li />', {'class': S.settings.className + '__item active'}) : $('<li />', {'class': S.settings.className + '__item'});
+        var $item     = (i === 0) ? $('<li />', {'class': S.settings.className + '__item' + S.settings.activeClass}) : $('<li />', {'class': S.settings.className + '__item'});
```

Comment:
Finally got around to testing this. There needs to be a space following `_item` otherwise it creates a single class as `scroll-nav__itemactive`.